modules/RTCStats/RTCStats.ts

```diff
@@ -0,0 +1,189 @@
+import rtcstatsInit from '@jitsi/rtcstats/rtcstats';
+import traceInit from '@jitsi/rtcstats/trace-ws';
+import EventEmitter from 'events';
+
+import { 
+    CONFERENCE_JOINED, 
+    CONFERENCE_LEFT, 
+    CONFERENCE_UNIQUE_ID_SET 
+} from '../../JitsiConferenceEvents';
+import JitsiConference from '../../JitsiConference';
+
+const RTC_STATS_WC_DISCONNECTED = 'rtcstats_ws_disconnected';
+const RTC_STATS_PC_EVENT = 'rtstats_pc_event';
+
+/**
+ * Filter out RTCPeerConnection that are created by callstats.io.
+ *
+ * @param {*} config - Config object sent to the PC c'tor.
+ * @returns {boolean}
+ */
+function connectionFilter(config) {
+    if (config?.iceServers[0] && config.iceServers[0].urls) {
```

Comment:
Is it enough to only took at the first ice server?

I think you can simplify the check with:

```
        for (const iceUrl of (config?.iceServers ?? [])[0]?.urls ?? []) {
```

Reply:
makes sense, changed.